tests/checker/good/gold/auction.scilla.typeinfo.gold

```diff
@@ -812,7 +812,7 @@
     },
     {
       "vname": "_sender",
-      "type": "ByStr20",
+      "type": "ByStr20 with end",
```

Comment:
Hmm, what does `ByStr20 with end` mean?

Reply:
It's the syntax of an address type. The general format is

```
Bystr20 with
  field_name : type,
  field_name : type,
  ...
end
```

A `ByStr20` with no declared fields just means that the address in question is in use, either as a user account or a contract.